challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,36 @@
+window.onload = getRandomCatPicture ()
+
+async function getRandomCatPicture () {
+    await fetch("https://cataas.com/cat?json=true")
+    .then(response => response.json())
+    .then(randomCat => {
+        const image = document.querySelector("img");
+        return image.src = "https://cataas.com" + randomCat.url;
```

Comment:
there is no `url` field in randomCat response ;) 
this api https://cataas.com/cat?json=true returns something like this:
```json
{
  "tags": [],
  "createdAt": "2022-06-13T14:12:07.461Z",
  "updatedAt": "2022-10-11T07:52:32.739Z",
  "mimetype": "image/jpeg",
  "size": 177034,
  "_id": "P9JloQHPE4Qlrf1Q"
}
```